x-pack/plugins/task_manager/server/plugin.ts

```diff
@@ -42,7 +42,7 @@ export class TaskManagerPlugin
       .toPromise();
 
     setupSavedObjects(core.savedObjects, this.config);
-    this.taskManagerId = core.uuid.getInstanceUuid();
+    this.taskManagerId = core.environment.getInstanceUuid();
```

Comment:
IMO it's confusing that `environment` service doesn't create `Env` public interface https://github.com/elastic/kibana/blob/85c8232c0b81e9e0dd2265e04b2cc26dffaf1b60/src/core/server/core_context.ts#L33
What if we make this service internal and pass `uuid` as a static value to the plugin constructor https://github.com/elastic/kibana/blob/fa93a81ba67f5177024f1ab3b4ac68919a7824dc/src/core/server/plugins/plugin.ts#L171 ? 

I understand that it's not required for the issue you are working on, so we can rollback all the changes to the `uuid` service.